Shaders/PBR/private/PBR_Textures.fxh

```diff
@@ -472,7 +472,9 @@ float3 SampleNormalTexture(PBRMaterialTextureAttribs TexAttribs,
 #endif
     }
 
-    return SampledNormal;
+    SampledNormal = normalize(SampledNormal * 2.0 - 1.0) * float3(TexAttribs.NormalScale, TexAttribs.NormalScale, 1.0);
+    SampledNormal = (SampledNormal + 1.0) / 2.0;
+    return float3(SampledNormal.rg, 1.0);
```

Comment:
`return float3(SampledNormal.rg, 1.0);`
This is plain wrong. By fixing z component at 1.0, you will only be able represent normals within a cone of size PI/4 around the normal. The furthest away you will be able to get is (-1, 0, 1) or (0, -1, 1)

Reply:
It's the least wrong answer I could come up with so far. :-) What it had going for it was matching output for my usecase with Blender, for what little that's worth.

Here's the discussion on the formula for the normal scale. Other sane options I've tried don't work either.

https://github.com/KhronosGroup/glTF/issues/885

I'm beginning to see why you might have skipped the scale portion of glTF

Reply:
Figured it out. There's no real need to put the normal back in the texture space [0,1] on return of the other functions. The other functions (MicroNormal and ClearCoat) return [-1, 1] normals anyway. 

Thanks for your patience.